source/whats-new.txt

```diff
@@ -57,7 +58,14 @@ The 3.0 driver release includes the following new features:
   This type is available in .NET 5 and later.
   To learn more about the ``Half`` type, see the
   `Half Struct <https://learn.microsoft.com/en-us/dotnet/api/system.half?view=net-8.0>`__
-  API reference page on MSDN. 
+  API reference page on MSDN.
+
+- The ``IMongoClient`` interface inherits the ``IDisposable`` interface. As a result, the
```

Comment:
Tech reviewer: I don't see this as a breaking change, but correct me if I'm wrong

Reply:
Yes, this is not a breaking change.
We should mention that `Dispose` disposes the underlying cluster and  connections to the server.
Also this is an experimental feature, so we should document it as such.